src/utils/aosheader.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef AOSHEADER_HPP
```

Comment:
copyright comments required.- and it better name the file as aos-protocol-header

Reply:
Done